docs/style-spec/expressions.md

```diff
@@ -0,0 +1,40 @@
+# Style Function Expressions
+
+**Constants:**
+- LN2
+- PI
+- e
+
+**Literal:**
+- JSON string / number / boolean literal
+
+**Property lookup:**
+- Feature property:  `{ "ref": "feature", "key": propertyName, "default": defaultValue}`
```

Comment:
I'm coming around to your original suggestion of just relying on the conditional operator to provide default behavior.